option optimize_for = LITE_RUNTIME;

message CSPointsRaceFightRequest
{
 required uint32 index = 1;
}
